Sources/ZMarkupParser/HTML/Processor/HTMLTagAttributeToMarkupStyleVisitor.swift

```diff
@@ -101,7 +101,7 @@ struct HTMLTagStyleAttributeToMarkupStyleVisitor: HTMLTagStyleAttributeVisitor {
     }
     
     func convert(fromPX string: String) -> Int? {
-        guard let regex = try? NSRegularExpression(pattern: "([0-9]+)p(x|t)"),
+        guard let regex = try? NSRegularExpression(pattern: "([0-9]+.?[0-9]*)p(x|t)"),
```

Comment:
Q: Should we change Int to CGFloat as well?

Reply:
Oh thank you, I forgot to change it to cgfloat, my case was only for 12.0px so I didn't think of that case. 
I was getting it as 0px in the 12.0px case.

I'll change it.